### PR TITLE
update operations manual to 2.8.2

### PR DESCRIPTION
**Pull request by monikavar**

### CODE REVIEW

File: docs/manual/readme.md [hunk 1]
```diff
@@ -750,43 +750,29 @@ Please note that we require Helm **3+**, and suggest **3.2+**, so be sure to set
...
 #### Installing the Chart
 
 ##### Add Helm Repository
 
 To install the chart first ensure that you have added either one of two Helm repositories where we publish azure-industrial-iot Helm chart. It can be found in:
 
-- <https://azure.github.io/Industrial-IoT/helm>
-
-  > NOTE: This is the recommended Helm chart repository to use.
+- <https://microsoft.github.io/charts/repo>
```
Comment (karok2m) [inline docs/manual/readme.md:761]: Please bring back details about `https://azure.github.io/Industrial-IoT/helm` Helm chart repository, preferably with the notes. Going forward this is the one that clients should use.

File: docs/manual/readme.md [hunk 2]
```diff
@@ -195,7 +195,7 @@ In this mode the OPC Publisher module runs a configurable number of workers (whi
 
 ##### Overview of how Standalone Mode Works
 
-OPC Publisher Module checks on a regular basis if there is a `publishednodes.json` file in its application directory. If there is one, the file is parsed and the OPC Publisher Module creates a new OPC UA subscription for each OPC UA server endpoint specified in the file and starts monitoring for data changes from this OPC UA server. Additionally, we support json schema validation of `publishednodes.json` file if a schema file is available to the OPC Publisher. Default location for schema file is `schemas/publishednodesschema.json` in its application directory. Default location of schema file can be changed via cli options [here](../modules/publisher-commandline.md), if a schema file is not found by OPC Publisher, a warning message is logged and module continues to load.
```
Comment (cristipogacean) [inline docs/manual/readme.md:198]: why do we want to remove the schema validation note here?

File: docs/manual/readme.md [hunk 3]
```diff
@@ -1545,9 +1531,6 @@ Several metrics are provided by default to give an overview of the health of the
 | iiot_edge_publisher_encoded_message_size_average      | deviceid, module, triggerid                      | Encoded IoT message body size average                                                   | gauge     |
 | iiot_edge_publisher_chunk_size_average                 | deviceid, module, triggerid                      | IoT Hub chunk size average                                                              | gauge     |
 | iiot_edge_publisher_estimated_message_chunks_per_day | deviceid, module, triggerid                      | Estimated IoT Hub chunks charged per day                                                | gauge     |
-| iiot_edge_publisher_is_connection_ok                 | deviceid, module, triggerid                      | Is the endpoint connection ok?                                                          | gauge     |
```
Comment (karok2m) [inline docs/manual/readme.md:1548]: Please add back details about `iiot_edge_publisher_is_connection_ok`, `iiot_edge_publisher_good_nodes` and `iiot_edge_publisher_bad_nodes` metrics. They are present in the current version.

File: docs/manual/readme.md [hunk 4]
```diff
@@ -930,8 +916,8 @@ The deployment script automatically configures all components to work with each
...
...al, Error, Warning, Information, Debug, Verbose).</td></tr><tr class="odd"><td>me|messageencoding</td><td>The messaging encoding for outgoing messages allowed values: Json, Uadp</td></tr><tr class="even"><td>mm|messagingmode</td><td>The messaging mode for outgoing messages allowed values: PubSub, Samples</td></tr><tr class="odd"><td>fm|fullfeaturedmessage</td><td>The full featured mode for messages (all fields filled in). Default is 'true', for legacy compatibility use 'false'</td></tr><tr class="even"><td>aa|autoaccept</td><td>The publisher trusted all servers it is establishing a connection to</td></tr><tr class="odd"><td>bs|batchsize</td><td>The number of OPC UA data-change messages to be cached for batching.</td></tr><tr class="even"><td>si|iothubsendinterval</td><td>The trigger batching interval in seconds.</td></tr><tr class="odd"><td>ms|iothubmessagesize</td><td>The maximum size of the (IoT D2C) message.</td></tr><tr class="even"><td>om|maxoutgressmessages</td><td>The maximum size of the (IoT D2C) message egress buffer.</td></tr><tr class="odd"><td>di|diagnosticsinterval</td><td>Shows publisher diagnostic info at the specified interval in seconds (need log level info).<br />
```
Comment (cristipogacean) [inline docs/manual/readme.md:933]: what was actually changed here? and what is the reason?

**Pull request closed without merging**